vendor/github.com/urfave/cli/app.go

```diff
@@ -467,6 +467,7 @@ func HandleAction(action interface{}, context *Context) (err error) {
 			switch r.(type) {
 			case error:
 				err = r.(error)
+				panic(err)
```

Comment:
no, no I did not. Wish there was a way to toggle this with --debug... it's a pain not having any idea where panic's come from.